testdata/case-0-error-microerror-Error-no-masking.golden

```diff
@@ -0,0 +1,5 @@
+{
+	"desc": "test-desc",
+	"docs": "test-docs",
+	"kind": "testKind"
+}
```

Comment:
Is it somehow vital that these don't have newlines? 
My editor will add them because it's a rebel and then I will have to deal with ensuring this.

Reply:
You shouldn't update this manually. This is produced by the golden test with `go test ./ -run Test_JSON -update`. See `Test_JSON` godoc.

Reply:
We can change the test to produce newlines if this is painful tho.

Reply:
You are right, it also prevents me from ever touching this manually.

Turns out I had it backwards - keep it :P

Reply:
I just pushed a "fix". Tell me whether I should keep it or not :)

Reply:
Keep it, all good :+1: